src/harness/sas_testcase.py

```diff
@@ -288,3 +291,26 @@ def TriggerFullActivityDumpAndWaitUntilComplete(self, server_cert, server_key):
       time.sleep(10)
     signal.alarm(0)
     return dump_message
+
+  def assertPpaWithinServiceArea(self, pal_records, ppa_zone_geometry):
```

Comment:
This is more appropriate in the PCR testcase file since it will be used only for PCR test cases.

Reply:
We have   moved this assert function to PCR testcase file based on your suggestion